src/app/Dependencies.tsx

```diff
@@ -0,0 +1,123 @@
+import { useState, useEffect, Fragment, FC } from 'react';
+import { jsx } from '@emotion/core';
+import ReactFlow, { ReactFlowProvider, isNode } from 'react-flow-renderer';
+import * as dagre from 'dagre';
+import { v4 as uuid } from 'uuid';
+import { getDependencyMap } from './commands';
+import { useStore } from './store';
+
+interface ExtensionItemProps {
+  dependencies: Record<string, string[]>;
+}
+
+const DisplayDependencies: FC<ExtensionItemProps> = ({ dependencies }) => {
+  const [elements, setElements] = useState([]);
+
+  useEffect(() => {
+    getData();
+  }, []);
+
+  const position = { x: 0, y: 0 };
+  const edgeType = 'smoothstep';
+  const dagreGraph = new dagre.graphlib.Graph();
+  dagreGraph.setDefaultEdgeLabel(() => ({}));
+
+  const nodeWidth = 172;
+  const nodeHeight = 36;
+
+  function getData() {
+    let nonDuplicatedDependencies: Array<string> = [];
+
+    Object.keys(dependencies).map(pilet =>
+      dependencies[pilet].map(dep => nonDuplicatedDependencies.push(dep.resolved || dep)),
```

Comment:
I think the TS interface still is not updated, right?

When something shows red squiggles please improve / fix it. Don't just ignore TS errors.